3rdparty/jvm/global_scalac_plugins.lock

```diff
@@ -0,0 +1,19 @@
+[
+    {
+        "coord": {
+            "group": "com.lihaoyi",
+            "artifact": "acyclic_2.13",
+            "version": "0.2.1",
+            "packaging": "jar"
+        },
+        "directDependencies": [],
```

Comment:
Obviously not for now or even maybe ever but camelCase snake_case mixing in the keys is twitch-inducing.

Reply:
Relatedly, probably better to add a versioning field for the lockfile format now rather than later.